cpp-fs/storage/storage/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+project(storage CXX)
+
+set(STORAGE_NAME storagelib)
+add_library(${STORAGE_NAME} storage.cpp)
+target_link_libraries(${STORAGE_NAME} PRIVATE Boost::headers tl::expected)
+target_include_directories(${STORAGE_NAME}
+  PUBLIC
+    ${CMAKE_CURRENT_LIST_DIR}
+    ${tl-expected_INCLUDE_DIR}
```

Comment:
This should be removed, let's instead rely on automatic include paths gathering with link-libraries facility